Bottom: e8223ea4b8860f521aaa28483504d75afbe9a667
Top:    263f2fbbbfc687ee902ccd5da4226645a11a9065
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-07 19:37:36 +0000

Refresh of 0001-lines-with-different-clearances-are-not-merged.patch

---

diff --git a/src/create.c b/src/create.c
index 0ee8697..7919b43 100644
--- a/src/create.c
+++ b/src/create.c
@@ -336,11 +336,11 @@ line_callback (const BoxType * b, void *cl)
       longjmp (i->env, 1);
     }
   /* remove unnecessary line points */
-  if (line->Thickness == i->Thickness
+  if (line->Thickness == i->Thickness &&
       /* don't merge lines if the clearances differ  */
-      && line->Clearance == i->Clearance
+      line->Clearance == i->Clearance &&
       /* don't merge lines if the clear flags differ  */
-      && TEST_FLAG (CLEARLINEFLAG, line) == TEST_FLAG (CLEARLINEFLAG, i))
+      TEST_FLAG (CLEARLINEFLAG, line) == TEST_FLAG (CLEARLINEFLAG, i))
     {
       if (line->Point1.X == i->X1 && line->Point1.Y == i->Y1)
 	{
